### ISSUE TITLE
Nested components cannot trigger click events

### ISSUE THREAD

**Issue by lc-c** [flame-engine/flame#635]
## Description

Nested components cannot trigger click events

## Development environment

Flame version: ^1.0.0-rc5

Flutter doctor:

```
 Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.5, on Linux, locale zh_CN.UTF-8)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[!] Android Studio (version 4.0)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] Connected device (1 available)

! Doctor found issues in 1 category.
```

## Runtime

This issue is related to running on which platform? (Select all that apply)

 - [x] Android :android:
 - [ ] iOS :apple:
 - [ ] Web :browser:

## Minimal reproducible code (Required for bugs)

Include a minimal code that reproduces the problem add it inline using code block markdown, or a link to Github repository:

**Comment by spydon** [comment]
@lc-c you'll have to include some code and more explanation to what is not working here, composed components and click event are working for me.

**Comment by lc-c** [comment]
game -> PositionComponent -> PositionComponent ->PositionComponent

**Comment by erickzanardo** [comment]
@lc-c as mentioned on the template, we need some [minimal reproducible code](https://stackoverflow.com/help/minimal-reproducible-example) when a bug is reported, could you please take a look on that so we can continue on analyzing this?

**Comment by spydon** [comment]
Fixed in `1.0.0-rc6`
